tox.ini

```diff
@@ -1,9 +1,16 @@
 [tox]
 envlist =
-    py{27,34,35}-sphinx{13,14,15}
+    py{27,34,35,36}-sphinx{13,14,15}
```

Comment:
I can't find an actual version range of Sphinx that they support, but I'd guess 1.3 isn't officially supported anymore.